src/utils/validate.ts

```diff
@@ -1,3 +1,5 @@
+import { getAddress } from 'ethers/lib/utils'
```

Comment:
```suggestion
import { getAddress } from '@ethersproject/address'
```

use `@ethersproject/*` packages instead of ethers